templates/tasks/scheduled_tasks/new_email_brandings.html

```diff
@@ -1,14 +1,87 @@
-<p>There are new email brands to review since {{ yesterday }}.</p><hr>
+{% if brands_by_organisation %}
+<h2>New email branding to review</h2>
+<p>Uploaded since {{ yesterday }}:</p>
+{% endif %}
 {%- for org, brands in brands_by_organisation.items() -%}
-    <p><a href="{{ domain }}/organisations/{{ org.id }}">{{ org.name }}</a>:</p><ul>
+    <p><a href="{{ domain }}/organisations/{{ org.id }}/settings/email-branding">{{ org.name }}</a>{% if not org.email_branding_id %} (no default){% endif %}:</p><ul>
     {%- for brand in brands -%}
         <li><a href="{{ domain }}/email-branding/{{ brand.id }}/edit">{{ brand.name }}</a></li>
     {%- endfor -%}
     </ul><hr>
 {%- endfor -%}
+{% if brands_by_organisation %}
+<h2>
+    How to review new email branding
+</h2>
+<p>
+    Make sure new email branding can remain in the branding pool. Check:
+</p>
+<ul>
+    <li>if it’s a duplicate</li>
+    <li>image quality</li>
+    <li>if it can be used by more than one service</li>
+    <li>branding name</li>
+</ul>
+
+<h3>1. Duplicates</h3>
+<p>
+    If the same logo is in the branding pool twice then keep whichever one looks best.
```

Comment:
If this happens, should any services with the 'worse' brand be moved over to the new one? Should we have guidance indicating to do or not do that?

Reply:
That’s the kind of quality control of individual services branding that we’re trying to get away from I think